src/main/java/com/fasterxml/jackson/databind/introspect/BeanPropertyDefinition.java

```diff
@@ -284,4 +286,15 @@ public String findReferenceName() {
      * @since 2.5
      */
     public abstract JsonInclude.Value findInclusion();
+
+    /**
+     * Method for finding all aliases of the property, if any.
+     *
+     * @return {@code List} of aliases, if any; never null (empty list if no aliases found)
+     *
+     * @since 2.17
+     */
+    public List<PropertyName> findAliases() {
```

Comment:
Decided to implement here for backwards compatibility (in case anything else extends base type)

Reply:
True... good catch